ansible/vtestbed.yaml

```diff
@@ -0,0 +1,62 @@
+---
+- conf-name: vms-kvm-t0
+  group-name: vms6-1
+  topo: t0
+  ptf_image_name: docker-ptf
+  ptf: ptf-01
+  ptf_ip: 10.250.0.102/24
+  ptf_ipv6: fec0::ffff:afa:2/64
+  server: server_1
+  vm_base: VM0100
+  dut:
+    - vlab-01
+  comment: Tests virtual switch vm
+- conf-name: vms-kvm-t0-64
+  group-name: vms6-1
+  topo: t0-64
+  ptf_image_name: docker-ptf
+  ptf: ptf-01
+  ptf_ip: 10.250.0.102/24
+  ptf_ipv6: fec0::ffff:afa:2/64
+  server: server_1
+  vm_base: VM0100
+  dut:
+    - vlab-02
+  comment: Tests virtual switch vm
+- conf-name: vms-kvm-t1-lag
+  group-name: vms6-2
+  topo: t1-lag
+  ptf_image_name: docker-ptf
+  ptf: ptf-02
+  ptf_ip: 10.250.0.106/24
+  ptf_ipv6: fec0::ffff:afa:6/64
+  server: server_1
+  vm_base: VM0104
+  dut:
+    - vlab-03
+  comment: Tests virtual switch vm
+- conf-name: vms-kvm-t0-2
+  group-name: vms6-3
+  topo: t0
+  ptf_image_name: docker-ptf
+  ptf: ptf-03
+  ptf_ip: 10.250.0.108/24
+  ptf_ipv6: fec0::ffff:afa:8/64
+  server: server_1
+  vm_base: VM0104
+  dut:
+    - vlab-04
+  comment: Tests virtual switch vm
+- conf-name: vms-kvm-dual-t0
```

Comment:
Would it be more readable if insert a blank line between each test setup?

Reply:
Updated, please take a review!